src/Tribe/Template.php

```diff
@@ -111,7 +111,7 @@ public function set_template_origin( $origin = null ) {
 			$this->origin = $origin;
 			$this->template_base_path = untrailingslashit( ! empty( $this->origin->plugin_path ) ? $this->origin->plugin_path : $this->origin->pluginPath );
 		} else {
-			$this->template_base_path = untrailingslashit( (array) explode( '/', $origin ) );
+			$this->template_base_path = untrailingslashit( (array) explode( '/', $origin ) ); // passing an array instead of a string?
```

Comment:
@bordoni `untrailingslashit()` requires a String, not an Array. What _should_ be here?

Reply:
Invert the logic, first `untrailingslashit` to after `explode`.